app/src/main/java/org/xbmc/kore/ui/BaseMediaActivity.java

```diff
@@ -48,6 +48,7 @@
 import org.xbmc.kore.jsonrpc.type.PlayerType;
```

Comment:
If we ditch the mechanism of opening the now playing panel, and always show the volume control dialog, this activity would also remain (almost) unchanged, which is a plus imo.